### ISSUE TITLE
OPenCV files from 2013

### ISSUE THREAD

**Issue by waltergrimmvie** [idpNET/secure-facial-recognition-security-system-using-emguCV#4]
Hi, 

worked straight of the box, 

questions: 

Why are you using opencv files from 2013 and not any nuget packages

Are there any advantages to this approach?

I have tried to google, but coud not find any answer on where to get the current updated native DLL files for OpenCV, any suggestions would be welcome

thanx

**Comment by idpNET** [comment]
> Hi,
> 
> worked straight of the box,
> 
> questions:
> 
> Why are you using opencv files from 2013 and not any nuget packages
> 
> Are there any advantages to this approach?
> 
> I have tried to google, but coud not find any answer on where to get the current updated native DLL files for OpenCV, any suggestions would be welcome
> 
> thanx

Greetings,

When it comes to acquiring updated native DLL files for OpenCV, the recommended approach is to directly download the latest version from the official OpenCV website or through the GitHub repository. You can either build OpenCV from source or utilize pre-built binaries provided by the OpenCV community.

In the case of the current project, I intentionally opted for the 2013 version because it was deemed suitable for the project's requirements and compatible with the environment in which I was working.

In summary, utilizing a different version of the OpenCV SDK (whether through NuGet packages or manually adding references) may necessitate some code adjustments. I aim to utilize the newest NuGet packages of OpenCV for subsequent updates.

Thank you for reaching out!
